dubbo-common/src/main/java/org/apache/dubbo/config/ConfigCenterConfig.java

```diff
@@ -149,7 +146,7 @@ public URL toUrl() {
         if (StringUtils.isEmpty(address)) {
             address = ANYHOST_VALUE;
         }
-        map.put(PATH_KEY, ConfigCenterConfig.class.getSimpleName());
+        map.put(PATH_KEY, ConfigCenterConfig.class.getName());
```

Comment:
why change PATH_KEY value to full class name ?

Reply:
I see that the path of `MetadataReport`'s url is full class name, and `RegistryService` is also like this
![image](https://user-images.githubusercontent.com/43363120/147081535-2b9dc3ae-982e-49db-b524-4af101cb6569.png)
![image](https://user-images.githubusercontent.com/43363120/147082485-5893dfcb-2c87-4515-a2f3-c6134aff9280.png)

Reply:
@AlbumenJ  PTAL the `PATH_KEY`